include/rocksdb/metadata.h

```diff
@@ -35,6 +36,8 @@ struct ColumnFamilyMetaData {
   std::string name;
   // The metadata of all levels in this column family.
   std::vector<LevelMetaData> levels;
+
+  std::vector<BlobMetaData> blob_files;
```

Comment:
Could you also add `blob_size` (total size of all blob files) and `blob_file_count` as discussed to be in sync with what we have for SSTs?